grpc-api/src/main/java/io/deephaven/grpc_api/barrage/BarrageServiceGrpcImpl.java

```diff
@@ -124,7 +124,7 @@ public SubscriptionObserver(final SessionState session, final StreamObserver<Inp
             this.myPrefix = "SubscriptionObserver{" + Integer.toHexString(System.identityHashCode(this)) + "}: ";
             this.session = session;
             this.listener = listenerAdapter.adapt(responseObserver);
-            session.manage(this); // attach the liveness of this subscription to the session
+            this.session.attachLivenessReferent(this);
```

Comment:
This object should be a `SingletonLivenessManager` instead of a `LivenessArtifact`.